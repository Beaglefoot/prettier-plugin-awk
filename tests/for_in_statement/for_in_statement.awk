{
  for (  x in arr  ) {
    print arr[x]
  }
}
