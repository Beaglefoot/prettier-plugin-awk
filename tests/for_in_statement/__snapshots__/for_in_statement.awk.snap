{
  for (  x in arr  ) {
    print arr[x]
  }
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{
    for (x in arr) { print arr[x] }
}
